tokio/src/runtime/mod.rs

```diff
@@ -334,7 +334,7 @@ impl Runtime {
     /// ```
     ///
     /// [mod]: index.html
-    /// [main]: ../../tokio_macros/attr.main.html
+    /// [main]: ../attr.main.html
```

Comment:
Can this not be a intra-rustdoc link too?

Reply:
I was not able to find a way to do it for now.

Reply:
Maybe it is possible since https://github.com/rust-lang/rust/commit/14986081355db0a2ae67df6a43dd9e6e360d718c.

Reply:
Other places in the codebase do the same, so lets stick with this.